src/main/java/tutorease/address/logic/commands/AddContactCommand.java

```diff
@@ -61,6 +65,14 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
+        if (model.hasSamePhone(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_PHONE);
+        }
+
+        if (model.hasSameEmail(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_EMAIL);
+        }
+
```

Comment:
Perhaps can separate these to adhere to SLAP.

Reply:
Fixed!